proc/name.c

```diff
@@ -325,14 +327,15 @@ int proc_create(int port, int type, int mode, oid_t dev, oid_t dir, char *name,
 	msg->i.create.type = type;
 	msg->i.create.mode = mode;
 	msg->i.create.dev = dev;
-	msg->i.create.dir = dir;
+	msg->oid = dir;
```

Comment:
Safer to use memcpy. Seem both previous version and line above had this problem before too